probes/utils/code_review.go

```diff
@@ -0,0 +1,51 @@
+// © 2023 Nokia
+// Licensed under the BSD 3 Clause license
+// SPDX-License-Identifier: BSD-3-Clause
+
+package utils
+
+import (
+	"github.com/ossf/scorecard/v4/finding"
+	"embed"
+	"fmt"
+)
+
+// returns the number of unique reviewers that aren't the changeset author
+func UniqueReviewers(authorLogin string, reviewers []string) int {

```

Comment:
all of these functions are only called by one probe. do they need to be in this util folder, instead of part of the the probe that needs it?

Reply:
refactored the probe now, deleted the unnecessary utils